examples/webview/pyproject.toml

```diff
@@ -53,6 +53,10 @@ build_gradle_dependencies = [
     "com.google.android.material:material:1.12.0",
 ]
 
+build_gradle_extra_content="""
+chaquopy.defaultConfig.staticProxy("toga_android.widgets.webview")
+"""
```

Comment:
Hrm... Requiring users to manually inject an "extra content" block to make a feature work is a problematic requirement. I understand why it's needed - but we possibly need to solve the bigger problem of allowing *toga* to declare a list of "classes that need a static proxy".

Reply:
I agree. But this probably means that this PR cannot be merged any time soon, right?

Reply:
It depends how essential the change is.

If a WebView will continue to work on Android, but this feature won't work without the extra content declaration, we can probably merge it. We will need to add a set of platform notes in the docs, and we'll need to prioritise finding a better fix - but I don't think it needs to be a complete blocker.

However, if Webview (or the whole Android backend) won't work *at all* if this addition isn't in place, then we'll need to solve that problem first.